TheJumpCloud/support
# Sa 2331 automate wiki docs generation

## Issues
* [SA-2331](https://jumpcloud.atlassian.net/browse/SA-2331) - Automate Wiki Docs Generation

## What does this solve?

When the docs for the powershell module are updated they should pushed to the support.wiki. This PR addresses the issue.

On the CI release task, a job should trigger to commit the latest docs to the support wiki repo now. 

## Is there anything particularly tricky?

No

## How should this be tested?

See this CI build:
https://app.circleci.com/pipelines/github/TheJumpCloud/support/764/workflows/647d92fd-b00a-4f4a-8c6b-e624a3543b75/jobs/2885

## Screenshots
